src/js/components/recipientLanding/RecipientLandingSearchBar.jsx

```diff
@@ -76,7 +76,7 @@ export default class RecipientLandingSearchBar extends React.Component {
                         name="recipient"
                         value={this.state.recipient}
                         onChange={this.onChange}
-                        placeholder="Search by Recipient Name or DUNS" />
+                        placeholder="Recipient Name, UEI, or DUNS" />
```

Comment:
placeholder should include the global constant before "DUNS"

placeholder=`Recipient Name, UEI, or ${GlobalConstants.DUNS_LABEL}DUNS` />